src/backend/scripts/setup_localdata.py

```diff
@@ -110,20 +112,35 @@ def create_gisaid(session):
         print("Aligned Gisaid Dump already exists")
         return
     # Add raw gisaid dump
-    gisaid_s3_bucket = "gisaid_bucket"
+    gisaid_s3_bucket = "genepi-gisaid-data"
+    s3_resource = boto3.resource(
+        "s3",
+        endpoint_url=os.getenv("BOTO_ENDPOINT_URL") or None,
+        config=boto3.session.Config(signature_version="s3v4"),
+    )
     suffix = datetime.now().isoformat()
+    raw_s3_key=f"raw_gisaid_dump-{suffix}"
+    processed_sequences_s3_key=f"processed_sequences-{suffix}"
+    processed_metadata_s3_key=f"processed_metadata-{suffix}"
+    aligned_sequences_s3_key=f"aligned_sequences-{suffix}"
+    aligned_metadata_s3_key=f"aligned_metadata-{suffix}"
     raw_gisaid_dump = RawGisaidDump(
         download_date=datetime.now(),
         s3_bucket=gisaid_s3_bucket,
-        s3_key=f"raw_gisaid_dump-{suffix}",
+        s3_key=raw_s3_key,
     )
     session.add(raw_gisaid_dump)
+    s3_resource.Bucket(gisaid_s3_bucket).Object(raw_s3_key).put(Body="")
+    s3_resource.Bucket(gisaid_s3_bucket).Object(processed_sequences_s3_key).put(Body="")
+    s3_resource.Bucket(gisaid_s3_bucket).Object(processed_metadata_s3_key).put(Body="")
+    s3_resource.Bucket(gisaid_s3_bucket).Object(aligned_sequences_s3_key).put(Body="")
+    s3_resource.Bucket(gisaid_s3_bucket).Object(aligned_metadata_s3_key).put(Body="")
```

Comment:
Actually write some gisaid files to s3 so our nextstrain scripts can download them in local dev.